package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "onchange",
-  "version": "6.0.0",
+  "version": "6.1.0",
```

Comment:
Please don't bump versions in a feature PR.

Reply:
ok, I revert in one next commit.